base/reinterpretarray.jl

```diff
@@ -136,3 +162,129 @@ end
     end
     return a
 end
+
+# Special case for StridedArray
+reinterpret_alignment(::Type{ReinterpretArray{T,N,S,A}} where {N, S, A<:Array}) where {T} = datatype_alignment(T)
+reinterpret_alignment(::Type{<:ReinterpretArray}) = gcd(datatype_alignment(T), datatype_alignment(S))
```

Comment:
What’s `T`, `S`?

Reply:
If this is indeed a bug (seems very likely), it would be good to have a test case that would catch it.